Main/ClientApp/src/Components/Drawer/MyCollectionsSection.tsx

```diff
@@ -111,6 +179,16 @@ export default function MyCollectionsSection() {
         toggleDialogOpen={() => setShareDialogOpen(!shareDialogOpen)}
         collectionId={selectedCollection}
       ></ShareCollectionDialog>
+      <DeleteCollectionDialog
+        open={deleteDialogOpen}
+        toggleDialogOpen={() => setDeleteDialogOpen(!deleteDialogOpen)}
+        confirmAction={() => deleteCollection(selectedCollection)}
+      />
+      <EditCollectionDialog
+        open={updateDialogOpen}
+        toggleDialogOpen={() => setUpdateDialogOpen(!updateDialogOpen)}
+        collectionId={selectedCollection}
+      />
```

Comment:
There is no consistency between the dialogs, one is getting confirmAction props, the other calls the action from itself. Consider unifying that so code doesn't become spaghetti